tests/integration_tests_control_files/reduced_order/inviscid_naca0012_reduced_order_consistency_snapshots.sh

```diff
@@ -76,7 +76,7 @@ echo "  end" >> $file
 echo "end" >> $file
 
 dir=$(pwd)
-mpirun "-n" "4" "$1/PHiLiP_2D" "-i" "${dir}/${file}"
+mpirun "-n" "$2" "$1/PHiLiP_2D" "-i" "${dir}/${file}"
```

Comment:
Does this file have access to MPIMAX? mpirun with "$2" probably runs it with all available processors, even when ${MPIMAX} is set to a lower value.

Reply:
Yes, the MPIMAX is passed to the script in CMakeLists.txt as the second argument, so $2 here contains the value for MPIMAX.

Reply:
Hmm ok, and can $2 be replaced with MPIMAX in this line?

Reply:
No it needs to be $2, since it is the second argument that I passed in CMakeLists.txt. For example, the CMake test has the command:

`./inviscid_naca0012_reduced_order_consistency_snapshots.sh ${EXECUTABLE_OUTPUT_PATH} ${MPIMAX}`

Therefore, in inviscid_naca0012_reduced_order_consistency_snapshots.sh, $1 is EXECUTABLE_OUTPUT_PATH and $2 is MPIMAX. These are called positional parameters in Bash.

Reply:
Oh ok, got it. Thanks.